### ISSUE TITLE
[Crash] Replanter crash / Incompatibility with Supplementaries

### ISSUE THREAD

**Issue by SirRosenfeld** [strikerrocker/VanillaTweaks#181]
The automatic seed-replanting system crashes the game when playing with Supplementaries.

Minecraft 1.19.2
Fabric 0.14.10
Supplementaries 2.2.23
Vanille Tweaks 1.5.63

Log error :
```
net.minecraft.class_148: Ticking entity
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:879) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819) ~[client-intermediary.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.IllegalArgumentException: Cannot get property class_2754{name=half, clazz=class net.minecraft.class_2756, values=[upper, lower]} as it does not exist in Block{minecraft:air}
	at net.minecraft.class_2688.method_11654(class_2688.java:98) ~[client-intermediary.jar:?]
	at net.mehvahdjukaar.supplementaries.common.block.blocks.FlaxBlock.method_9558(FlaxBlock.java:90) ~[supplementaries-1.19.2-2.2.23-fabric-WLTDCustom.jar:?]
	at net.minecraft.class_1542.handler$fgo009$tick(class_1542.java:3584) ~[client-intermediary.jar:?]
	at net.minecraft.class_1542.method_5773(class_1542.java:103) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_18762(class_3218.java:712) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_18472(class_1937.java:466) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_31420(class_3218.java:371) ~[client-intermediary.jar:?]
	at net.minecraft.class_5574.method_31791(class_5574.java:54) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_18765(class_3218.java:339) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:875) ~[client-intermediary.jar:?]
	... 5 more
```

Crash report :
[crash-2022-11-30_15.25.53-server.txt](https://github.com/StrikerRockers-Mods/VanillaTweaks/files/10124178/crash-2022-11-30_15.25.53-server.txt)

**Comment by sargeow** [comment]
Hello,

Talked to the Supplementaries mod dev and said this was a vanilla tweaks issue. They said the bug has been present for a year or so now. "they are giving the flax block a block state that's not its own breaking the methods contract" -Supplementaries mod dev

[https://pastebin.com/dWLVt4Gf]- CLIENT
[https://pastebin.com/QUWqYK0R] - SERVER

Thanks!

**Comment by strikerrocker** [comment]
Fixed the issue by adding a blacklist containing flax seeds from supplementaries by default. Also released a fix for 1.19.2 also,
